apps/frontend/src/components/global/checklist/ChecklistSeverityOverride.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <v-card class="text-center px-8 pt-2" height="300px">
+    <v-card-title class="justify-center"
+      >Severity Override Justification</v-card-title
+    >
+    <v-card-subtitle
+      v-if="selectedRule.severityJustification === ''"
+      class="justify-center mt-1"
+    >
+      <strong>
```

Comment:
seems like the 'strong' doesn't actually impact how it looks.  if we need it to be larger or something then we should do it via css unless there's a semantic reason why we need 'strong'.

Reply:
Removed